pkg/controller/codestatus/show.go

```diff
@@ -18,18 +18,37 @@ package codestatus
 
 import (
 	"net/http"
+	"time"
 
 	"github.com/google/exposure-notifications-verification-server/pkg/controller"
 )
 
-func (c *Controller) Show() http.Handler {
+func (c *Controller) HandleShow() http.Handler {
 	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
 		ctx := r.Context()
 
+		uuids := r.URL.Query()["uuid"]
```

Comment:
We should either:

- (preferred) make the form a POST and then use `FormData` with `controller.BindForm` here
- Make the entire thing a GET request and put the UUID in the URL (not really preferred because it leaks the UUIDs in server logs)